assets/javascripts/discourse/templates/connectors/after-reviewable-flagged-post-body/sift.hbs

```diff
@@ -0,0 +1,30 @@
+<div class="sift-classification">
```

Comment:
This is great that we have a template plug in again, although I think being able to override the text body would fit better with what this particular client wants.  They would like to show the text and the tokenization/risk analysis together, which is what you provided with the initial Reviewable code (with the custom reviewable where we had full control of the template) but that leads to the problem of having to review 2 different objects if a post if flagged by both Sift and a user, which this client definitely does not want :-)
This is more than sufficient as long as we can still add our action buttons (and also be able to call the standard discourse actions from within our custom actions, which I think you implied was possible in an email?  i.e. we want to add back our report buttons like "Agree with classification" which will call a sift endpoint for reporting, but will also need to call the default Discourse action `flag_agreed` because currently they have to click our button *and* click the standard Discourse review buttons.  We just don't know how to do that, but I am assuming is a standard type of operation in Rails and/or Ember?  We could also do it if we can "hook" to the standard actions to also call our own code, then we wouldn't add our own buttons.
Examples of the action buttons are found in `assets/javascripts/discourse/templates/connectors/flagged-post-below-controls/sift-flag-queue.hbs`

Reply:
What I mentioned over email is exactly the second option - that you could 'hook' into the standard actions to call additional functions, so no additional buttons are added.

The template here does add the tokenization/risk analysis - both the sift reviewables and the flagged user reviewables look identical (action buttons aside) with this:

<img width="617" alt="Screen Shot 2019-11-13 at 2 05 13 PM" src="https://user-images.githubusercontent.com/1322534/68808461-a4afed80-061e-11ea-8e00-7a8cd8335039.png">

Reply:
That great news on being able to hook into the standard action, we just might need some hints on how to do it later :-)

With the template, I just meant that with the custom reviewable we have control of the template and so can just show the tokenization, whereas with the outlet it will be the same as the current standard queue, where the post text is above (from the standard Discourse template) and our tokenization is below, and so basically repeats the post text   What the client actually wants is to show the tokenization inline with the text.  Your screenshots don't quite show the issue, as the text is different (because the Sift response is coming from the fixture) but in practice the 2 sets of text are the same (except for the classification badging)  It is more obvious with longer text

Reply:
Oh I see - we can probably improve that a little bit with either CSS hiding for the post content, or updating how the serializers return data (so that the "post" is blank if there exists classification data, something like that)